src/lib/utility.js

```diff
@@ -60,9 +62,15 @@ module.exports = {
         } else {
           rippleds[p_pk] = {ipp: p_ipp, version: p_v, uptime: uptime};
         }
-
       });
     });
+    // correcting uptime once rippleds and maxUptimeByIpp are ready
+    _.each(rippleds, function(r) {
+      var peerMaxUptime = r.ipp && maxUptimeByIpp[r.ipp];
```

Comment:
yup, it's a guard operator :) usually if I have a deeply nested property, i do
var value = foo && foo.bar && foo.bar.value, 
to guard against undefined's